How to run the minimal test

Hi, dumb question, but I'm trying to learn how to run ROS tests and was wondering how to actually run this. I did 

``ros2 test src/minimal_integration_test/test/launch/basic_test.test.py``

but I got an error 

``[ERROR] [launch]: Caught exception in launch (see debug for traceback): Included launch description missing required argument 'test_binary_dir' (description: 'Binary directory of package containing test executables'), given: []``

Thanks!

--- a/README.md
+++ b/README.md
@@ -1,0 +1,13 @@
+## Minimal Integration Test
+
+To build the test:
+
+```
+colcon build
+```
+
+To run the test:
+
+```
+colcon test --packages-select minimal_integration_test
+```

